src/orgHooks.ts

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
org hook stuff is moving to `plugin-org`, right?

Reply:
yes!